features/networking/ovn.feature

```diff
@@ -0,0 +1,197 @@
+Feature: OVN related networking scenarios
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-28936
+  @admin
+  @destructive
+  Scenario: Create/delete pods while forcing OVN leader election
+  #Test for bug https://bugzilla.redhat.com/show_bug.cgi?id=1781297
+    Given the env is using "OVNKubernetes" networkType
+    Given I have a project
+    And evaluation of `project.name` is stored in the :usr_project clipboard
+
+    Given I run the steps 4 times:
+    """
+    Given I have a pod-for-ping in the "<%= cb.usr_project %>" project
+    Given I store the ovnkube-master "south" leader pod in the clipboard
+    When admin deletes the ovnkube-master "south" leader
+    Then the step should succeed
+    When I store the ovnkube-master "south" leader pod in the :new_south_leader clipboard
+    Then the step should succeed
+    And the expression should be true> cb.south_leader.name != cb.new_south_leader.name
+    And admin waits for all pods in the "openshift-ovn-kubernetes" project to become ready up to 60 seconds
+    Given I ensure "hello-pod" pod is deleted from the "<%= cb.usr_project%>" project
+    """
+
+
+  # @author rbrattai@redhat.com
+  # @case_id OCP-26092
+  @admin
+  @destructive
+  Scenario: Pods and Services should keep running when a new raft leader gets be elected
+    Given the env is using "OVNKubernetes" networkType
+    Given I store the ovnkube-master "south" leader pod in the clipboard
+    Given I have a project
+    And evaluation of `project.name` is stored in the :usr_project clipboard
+    When I run the :create client command with:
+      | f | <%= BushSlicer::HOME %>/testdata/networking/list_for_pods.json |
```

Comment:
Could you try obtain test data file style ? Examples in https://github.com/openshift/verification-tests/pull/1227/files

The idea is that we do not want to rely on specific test data path.

Reply:
done